pages/projects/[id]/tasks/[taskId]/index.js

```diff
@@ -34,16 +31,21 @@ export default function TaskDetailPage() {
     error,
   } = useSWR(`/api/projects/${id}/tasks/${taskId}`);
 
-  if (!isReady || isLoading) return <h2>Loading..</h2>;
+  if (!isReady || isLoading) return <Loading />;
   if (error) return <Error message={error.message} />;
 
   const { title, description, tag, createdAt } = task;
+  const creationDate = createdAt.substr(0, 10);
```

Comment:
Have a look at [Date.prototype.toLocaleDateString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) ^^

[For more date related stuff](https://codesandbox.io/s/dealing-with-dates-8vlgth?file=/README.md)

Reply:
Right, of course!